python/amici/__init__.py

```diff
@@ -223,3 +226,28 @@ def runAmiciSimulations(model, solver, edata_list, failfast=True,
                                                    failfast,
                                                    num_threads)
     return [numpy.ReturnDataView(r) for r in rdata_ptr_list]
+
+
+def readSolverSettingsFromHDF5(*args) -> None:
```

Comment:
why not explicit arguments? + docstring what is to be passed?

Reply:
Hmm, the function is overloaded (allows passing file either as `str` or `H5::H5File`). In the RTD documentation this is properly processed but looks like tools such as PyCharm do indeed not have proper typhints+docstrings. Will fix as the arguments are the same across overloads.